packages/react-flicking/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "react-flicking",
+  "version": "0.1.0",
```

Comment:
@daybrush we've concluded that react-flicking starts from v3. If some changes are expected, it would be better to add postfix '-beta'.